code/src/WijmoProvider/Columns/CalculatedColumn.ts

```diff
@@ -4,8 +4,8 @@ namespace WijmoProvider.Column {
         constructor(
             grid: OSFramework.Grid.IGrid,
             columnID: string,
-            configs: JSON,
-            extraConfig: JSON
+            configs: any,
```

Comment:
changing from JSON to any will trigger lint errors right? Do we need to change to any in the first place?

Reply:
It doesn't trigger errors. I changed to any, because of the following image.

![image](https://user-images.githubusercontent.com/3113318/128706570-a72be994-abb6-4009-a454-862d79a80fed.png)

Reply:
It might have been a better approach using a config file, if we end up making changes on SS side as this might break if we don't remember that we have to change it here too. But that's just a comment, it seems good